core-bundle/src/EventListener/PrettyErrorScreenListener.php

```diff
@@ -132,32 +138,33 @@ private function renderErrorScreenByType(int $type, ExceptionEvent $event): void
 
         $processing = true;
 
-        if (null !== ($response = $this->getResponseFromPageHandler($type))) {
-            $event->setResponse($response);
-        }
-
-        $processing = false;
-    }
+        try {
```

Comment:
The try block seems to be longer than it should be. Is it not just required around the `$this->httpKernel->handle()` method?

Reply:
changed in 2752e29fc707bce414fbb0e41a3f2b0980a30c75